tests/TestCase.php

```diff
@@ -24,6 +24,10 @@ public function setUp(): void
     {
         parent::setUp();
 
+        if (isset($this->minimumLaravelVersion)) {
```

Comment:
We don't have a lot of tests that need this kind of check, so let's move in the file where we need it.

Reply:
👍